### ISSUE TITLE
Asana Oauth redirecting to http://localhost:10639/oauth_callback

### ISSUE THREAD

**Issue by mLuby** [mLuby/motivatr#6]
```
Error
invalid_request: The `redirect_uri` parameter does not match a valid url for the application.
```

URL: https://app.asana.com/-/oauth_authorize?client_id=43511055636261&response_type=code&redirect_uri=http%3A%2F%2Flocalhost%3A10639%2Foauth_callback&scope=default

**Comment by mLuby** [comment]
<img width="513" alt="screen shot 2015-08-02 at 08 40 15" src="https://cloud.githubusercontent.com/assets/2483420/9026799/20a7a850-38f2-11e5-91df-a64f7d282c32.png">

**Comment by mLuby** [comment]
That port is the heroku port. 

```
2015-08-02T15:37:05.183628+00:00 app[web.1]: Listening on port 10639
```

**Comment by mLuby** [comment]
now set to `https://motivatr-app.herokuapp.com:39685/oauth_callback`

**Comment by mLuby** [comment]
Needed correct redirectUri without port.
Fixed by 3f7f3be77b2a81f11c1028695dca6c8b0abc43be and b4adc960300a234080cb671055321f4ec231dcd7.
